test/runtests.jl

```diff
@@ -137,5 +137,5 @@ close(dir)
 
 
 if !Debug
-	run(`rm -rf $tmp`)
+    run(`rm -rf $tmp`)
```

Comment:
Given that there's a debug option to disable this, I'll just switch to `rm` function.